lib/src/__testutils__/rtlUtils.tsx

```diff
@@ -27,3 +29,18 @@ type RenderElementRetval<P> = (additionalProps?: Partial<P>) => HTMLElement | un
 export function rtlRenderElement<P>(Component: ComponentType<P>, minimalProps: P): RenderElementRetval<P> {
   return pipe(rtlRender(Component, minimalProps), path(['container', 'firstElementChild']));
 }
+
+export async function runTimers() {
```

Comment:
I just remembered Combobox also has these functions defined, which can be removed & imported from `rtlUtils.tsx`

Reply:
> I just remembered Combobox also has these functions defined, which can be removed & imported from rtlUtils.tsx

Thanks for catching these. f16e26b7a